pandas/tests/extension/json/test_json.py

```diff
@@ -71,3 +71,6 @@ def test_value_counts(self, all_data, dropna):
 
 class TestCasting(base.BaseCastingTests):
     pass
+
+# We intentionally don't run base.BaseSetitemTests because pandas'
+# internals has trouble setting sequences of values into scalar positions.
```

Comment:
you can also add the test class as above, but skip the full class (I think the pytest skip decorators also work for classes)

Reply:
I've been scared off mixing inheritance and skipping classes since seeing pytest-568, where skipping in a child marks all the other children as skips.

Reply:
that's a good reason :-)